### PR TITLE
Added a recipe for Flask-Admin

### PR DESCRIPTION
**Pull request by pmlandwehr**
`Flask-Admin` is a very flexible package, with many modules that have outside dependencies but, at the same time, aren't likely to be used. As such, I've left those dependencies out but provided some guidance within the recipe itself. The documentation is obviously the definitive source.

### CODE REVIEW

File: recipes/flask-admin/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    # - shapely
+    # Required for flask_admin.contrib.mongoengine:
+    # - flask-mongoengine
+    # Required for flask_admin.contrib.peewee and flask_admin.contrib.peeweemodel:
+    # - peewee
+    # - wtf-peewee
+    #  Required for flask_admin.contrib.pymongo:
+    # - pymongo
+    # Required for flask_admin.contrib.sqla and flask_admin.contrib.sqlamodel:
+    # - sqlalchemy
```
Comment (ocefpaf) [inline recipes/flask-admin/meta.yaml]: This comments are optional dependencies? Do you want to provide them or are you OK with this PR the way it is?

Comment (pmlandwehr) [inline recipes/flask-admin/meta.yaml]: I'm ok with the way it is; this is a pretty diverse set of dependencies that fit a variety of use cases; there are plenty of folks who might want to use `Flask-Admin` without using Google AppEngine or MongoDB.

File: recipes/flask-admin/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - flask >=0.7
+    - wtforms
```
Comment (jakirkham) [inline recipes/flask-admin/meta.yaml]: Do we need all of these at build time or can some of them be dropped?

File: recipes/flask-admin/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/flask-admin/meta.yaml]: Maybe related to issue ( https://github.com/conda/conda/issues/3299 ). Though some people have already stated they think this case is out of scope. That being said, I don't think a final decision has occurred on that yet.

**Comment by jakirkham** [comment]
Thanks @pmlandwehr .
